virttest/utils_conn.py

```diff
@@ -1299,6 +1299,10 @@ def server_setup(self, on_local=False):
                         cakey_path: self.pki_CA_dir,
                         servercert_path: self.libvirt_pki_dir,
                         serverkey_path: self.libvirt_pki_private_dir}
+            cmd = "mkdir -p {0}".format(self.pki_CA_dir)
```

Comment:
whether put those logic beside line 1290, where cmd = "mkdir -p %s" % self.libvirt_pki_private_dir

Reply:
We only need to create this file before copying it to the remote host, so I think we should keep it.